src/main/java/edu/umich/ctools/esb/utils/WAPI.java

```diff
@@ -0,0 +1,213 @@
+package edu.umich.ctools.esb.utils;
+
+import java.util.HashMap;
+
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.apache.commons.codec.binary.Base64;
+import org.json.JSONObject;
+
+import com.mashape.unirest.http.HttpResponse;
+import com.mashape.unirest.http.JsonNode;
+import com.mashape.unirest.http.Unirest;
+import com.mashape.unirest.http.exceptions.UnirestException;
+
+public class WAPI 
+{
+	private static Log M_log = LogFactory.getLog(WAPI.class);
+
+	private final int HTTP_SUCCESS = 200;
```

Comment:
May want these to be public.  Other code might want to use them since they will be looking at the results of the call.